### PR TITLE
LPS-121870

### PR DESCRIPTION
**Pull request by jonmak08**
Hey @jonmak08,

Attached is an update for http://issues.liferay.com/browse/LPS-121870.

Please let me know if you have any questions. Thanks!

### CODE REVIEW

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

**Comment by jonmak08** [comment]
ci:test:sf

**Comment by jonmak08** [comment]
ci:test:relevant

**Comment by jonmak08** [comment]
/cc @markocikos, @julien, @ambrinchaudhary

Here is where I am pushing all my source formatting nits for the MT migrations and any extra minor fixes for the "closed" JIRA tickets that we have already completed.  These are all minor fixes, but just to shed some light into what I am doing/reviewing from our migration.

Please let me know if you have any questions. Thanks!

**Comment by jonmak08** [comment]
If these tests pass I will forward them along tomorrow and continue working through the rest of the completed/closed JIRA ticket components for our migrations.

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/js/BlogImagesManagementToolbarPropsTransformer.js [hunk 1]
```diff
@@ -23,25 +23,27 @@ export default function propsTransformer({portletNamespace, ...otherProps}) {
...
 			const form = document.getElementById(`${portletNamespace}fm`);
 
-			if (form) {
-				var cmd = form.querySelector(`#${portletNamespace}cmd`);
+			if (!form) {
+				return;
+			}
 
-				cmd?.setAttribute('value', 'delete');
+			var cmd = form.querySelector(`#${portletNamespace}cmd`);
```
Comment (julien) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/js/BlogImagesManagementToolbarPropsTransformer.js:30]: Shouldn't this be `const`?

File: modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/js/BlogImagesManagementToolbarPropsTransformer.js [hunk 2]
```diff
@@ -23,25 +23,27 @@ export default function propsTransformer({portletNamespace, ...otherProps}) {
...
+			cmd?.setAttribute('value', 'delete');
 
-				deleteFileEntryIds?.setAttribute(
-					'value',
-					Liferay.Util.listCheckedExcept(
-						form,
-						`${portletNamespace}allRowIds`
-					)
-				);
+			var deleteFileEntryIds = form.querySelector(
```
Comment (julien) [inline modules/apps/blogs/blogs-web/src/main/resources/META-INF/resources/blogs_admin/js/BlogImagesManagementToolbarPropsTransformer.js:34]: Same comment as above, shouldn't this be `const`

**Review by ambrinchaudhary — APPROVED** [review]

**Comment by jonmak08** [comment]
New Pull Request: https://github.com/jonmak08/liferay-portal/pull/2136

**Pull request closed without merging**